frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -653,6 +645,16 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
     verb: 'patch',
     name: cv.metadata.name,
   });
+  if (new URLSearchParams(window.location.search).has('showVersions')) {
+    clusterUpdateModal({ cv })
+      .then(() => removeQueryArgument('showVersions'))
+      .catch(_.noop);
+  }
+  if (new URLSearchParams(window.location.search).has('showChannels')) {
```

Comment:
I missed this on previous review, but I'd suggest an `else if` here to prevent ever opening two dialogs (if the URL has both query parameters for whatever reason).

Reply:
Updated.  One more tag, @spadgett.